baseplate/events/queue.py

```diff
@@ -26,12 +26,15 @@
 from thrift import TSerialization
 from thrift.protocol.TJSONProtocol import TJSONProtocolFactory
 
-from . import MAX_EVENT_SIZE, MAX_QUEUE_SIZE
 from baseplate.context import ContextFactory
 from baseplate.message_queue import MessageQueue, TimedOutError
 from baseplate._utils import warn_deprecated
```

Comment:
Same deal, maybe we should take this opportunity to have these files import the same way as other files.